android-app/app/src/androidTest/java/de/dhbw/ka/se/fibo/TestMatchers.java

```diff
@@ -1,13 +1,35 @@
 package de.dhbw.ka.se.fibo;
 
 import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
 
 import com.google.android.material.textfield.TextInputLayout;
 
+import org.hamcrest.Description;
 import org.hamcrest.Matcher;
 import org.hamcrest.TypeSafeMatcher;
 
 public class TestMatchers {
+
+    public static Matcher<View> childAtPosition(
+            final Matcher<View> parentMatcher, final int position) {
```

Comment:
```suggestion
            Matcher<View> parentMatcher, int position) {
```

Does not need to be final